### PR TITLE
Use execFileSync in typing installer

### PR DESCRIPTION
**Pull request by sheetalkamat**

### CODE REVIEW

File: src/typingsInstallerCore/typingsInstaller.ts [hunk 1]
```diff
@@ -31,28 +31,35 @@ namespace ts.server.typingsInstaller {
...
+            hasError = install(result.command[0], result.command[1]) || hasError;
         }
         return hasError;
     }
 
+    function getUserAgent(tsVersion: string) {
+        return `--user-agent="typesInstaller/${tsVersion}"`;
+    }
+    const npmInstall = "install", ignoreScripts = "--ignore-scripts", saveDev = "--save-dev";
+    const commandBaseLength = npmInstall.length + ignoreScripts.length + saveDev.length + getUserAgent("").length + 5;
```
Comment (weswigham) [inline src/typingsInstallerCore/typingsInstaller.ts]: Should just say `getUserAgent("0.0.0")` rather than adding the `+ 5`, IMO.

Comment (sheetalkamat) [reply src/typingsInstallerCore/typingsInstaller.ts]: + 5 is for the spaces around them (the tsversion length is added later on while computing actual command length as we use to.

Comment (weswigham) [reply src/typingsInstallerCore/typingsInstaller.ts]: Ah. That was not immediately clear, sorry.

**Comment by sheetalkamat** [comment]
Ping @RyanCavanaugh

**Pull request merged**